airbyte-integrations/connectors/source-s3/Dockerfile

```
@@ -17,5 +17,5 @@ COPY source_s3 ./source_s3
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.32
+LABEL io.airbyte.version=0.1.33
```

```suggestion
LABEL io.airbyte.version=1.0.0
```
I think this change deserves a major version bump as it might introduce a breaking change: the discovery logic is simplified and running `discover` on `0.1.32` vs current version can lead to different catalogs.

---

changed to 1.0.0

---

changed to 1.0.0